Formula/armadillo.rb

```diff
@@ -34,6 +34,6 @@ def install
       }
     EOS
     system ENV.cxx, "test.cpp", "-I#{include}", "-L#{lib}", "-larmadillo", "-o", "test"
-    assert_equal Utils.popen_read("./test").to_i, version.to_s.to_i
+    assert_equal Utils.safe_popen_read("./test").to_i, version.to_s.to_i
```

Comment:
Any cases in the `test do` block should be `shell_output` instead (which will produce nicer error messages for tests). This could perhaps be an extension of the rubocop?